walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,123 @@
 
 package com.google.sps;
 
+import java.util.Arrays;
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public static final int DAY_START = TimeRange.START_OF_DAY;
+    public static final int DAY_END = TimeRange.END_OF_DAY;
+    public static final TimeRange WHOLE_DAY = TimeRange.WHOLE_DAY;
```

Comment:
Just my personal preferences and you don't have to change the code.
I won't define another set of variables for DAY_START/DAY_END/WHOLE_DAY since using TimeRange.XXX is already very clear and redefining seems a bit redundant.

Reply:
Ok a will-do for me! I just thought that having them defined as constants would make it clearer...

Reply:
I think either way is fine. Just if it's a valid request, then the function should return all free slots, right?
I agree with you and prefer to treat it as invalid request.
Marking as resolved.

Reply:
Sometimes is while sometimes not. In this case even the names didn't change much. So I'd prefer to avoid redefining.

Reply:
I also share Ying's perspective that redefining these constant locally doesn't add much value.

Reply:
The local constants are removed! :)